src/targets/enemies/al-kharid-warrior.js

```diff
@@ -0,0 +1,49 @@
+import { Enemy } from "./enemy.js";
+import NormalBones from "../../items/bones/normal-bones.js";
+import BronzeDagger from "../../items/equipment/daggers/bronze-dagger.js";
+
+export class AlKharidWarrior extends Enemy {
+    // Base stats
+    attack = 7;
+    strength = 5;
+    defense = 4;
+    magic = 1;
+    ranged = 1;
+
+    // Offensive bonuses
+    attackBonus = 10;
+    strengthBonus = 9;
+
+    // Defense
+    stabDefense = 12;
+    slashDefense = 15;
+    crushDefense = 10;
+    magicDefense = -1;
+    rangedDefense = 12;
+
+    // Text
+    examineText = "Part of Al Kharid's elite fighting force.";
+
+    constructor(scene) {
+        super({
+            scene: scene,
+            name: "Al Kharid Warrior",
+            varName: "alKharidWarrior",
+            images: [
+                {
+                    name: "al-kharid-warrior",
+                    path: "src/assets/sprites/AlKharidWarrior.png",
+                    scale: 0.3,
+                },
+            ],
+            offsetY: 50,
+            maxHealth: 19,
+            killGold: 20,
+            drops: [
+                { item: NormalBones, rate: 0.5 },
+                // { item: BronzeMedHelm, rate: 1 / 64 },
```

Comment:
Left this commented out? and "Replace with"* in below comment

Reply:
Bronze med helm doesn't exist yet, was just leaving it here for later whenever it's implemented, can remove if you want

Reply:
https://github.com/EthanVieira/rs-clicker/pull/88/commits/5bb0f8f3e5eafd9655948f42b769a8b65397bd39